hedera-node/src/main/java/com/hedera/services/ledger/MerkleLeafChanges.java

```diff
@@ -0,0 +1,19 @@
+package com.hedera.services.ledger;
+
+import com.hedera.services.ledger.properties.BeanProperty;
+
+import java.util.Map;
+
+/**
+ * Represents a set of changes to a merkle leaf.
```

Comment:
In hindsight, it seems to make more sense to name this class `LedgerEntityChanges`, and document it as representing "a set of changes to an entity belonging to a `TransactionalLedger`".